daemon/algod/api/server/v2/dryrun_test.go

```diff
@@ -1251,28 +1251,44 @@ func TestDryrunCost(t *testing.T) {
 		msg       string
 		numHashes int
 	}{
-		{"REJECT", 12},
-		{"PASS", 5},
+		{"REJECT", 22},
+		{"PASS", 16},
 	}
 
 	for _, test := range tests {
 		t.Run(test.msg, func(t *testing.T) {
-			costs := make([]uint64, 2)
+			costs := make([]int64, 3)
 
 			ops, err := logic.AssembleString("#pragma version 5\nbyte 0x41\n" + strings.Repeat("keccak256\n", test.numHashes) + "pop\nint 1\n")
 			require.NoError(t, err)
-			approval := ops.Program
-			costs[0] = 3 + uint64(test.numHashes)*130
+			app1 := ops.Program
+			costs[0] = 3 + int64(test.numHashes)*130
 
 			ops, err = logic.AssembleString("int 1")
 			require.NoError(t, err)
 			clst := ops.Program
 
 			ops, err = logic.AssembleString("#pragma version 5 \nint 1 \nint 2 \npop")
 			require.NoError(t, err)
-			approv := ops.Program
+			app2 := ops.Program
 			costs[1] = 3
 
+			ops, err = logic.AssembleString(`#pragma version 6
+itxn_begin
+int appl
+itxn_field TypeEnum
+int DeleteApplication
+itxn_field OnCompletion
+byte 0x068101
```

Comment:
You can add a comment inline here right? if so can you please just add `#pragma version 6; int 1;`